build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.0-rc01'
+        classpath 'com.android.tools.build:gradle:4.0.1'
```

Comment:
revert changes in thsi file

Reply:
After reverting the changes : 
Build failed due to: The project is using an incompatible version (3.6.0-rc01) of the Android Gradle plugin.